### PR TITLE
Address feedback related to GHA runners documentation

### PR DESCRIPTION
**Pull request by idelvall**

### CODE REVIEW

File: docs/cloud/satellites/gha-runners.md [hunk 1]
```diff
@@ -1,66 +1,48 @@
-# Satellites as GHA runners (**experimental**)
+# satellites as GitHub Actions runners (**experimental**)
```
Comment (vladaionescu) [inline docs/cloud/satellites/gha-runners.md:1]:
```suggestion
# Satellites as GitHub Actions runners (**experimental**)
```

File: docs/cloud/satellites/gha-runners.md [hunk 2]
```diff
@@ -1,66 +1,48 @@
...
-- `administration:write`
+{% hint style='info' %}
+Follow the [official docs](https://docs.github.com/en/authentication/keeping-your-account-and-data-secure/managing-your-personal-access-tokens) for detailed information on how to create a GitHub token, and make sure to set an expiration long enough, since the integration won't work after the token expires. 
+{% endhint %}
 
-### CLI
+### 3. Register via CLI
+Create the integration using the `earthly github add` CLI command, passing the token created in the previous step. 
 
 `$ earthly github add --help`
```
Comment (vladaionescu) [inline docs/cloud/satellites/gha-runners.md:35]: This should be the command the user actually runs. I would avoid just pasting the help text, and instead rewriting the help text here to explain it better. For example:

This command sets the configuration to create a new GitHub-Earthly integration, to trigger satellite builds from GitHub Actions (GitHub Actions).

```
earthly github add --org <earthly-org> --gh-org <github-org> --gh-repo <github-repo> --gh-token <token>
```

Where
* `earthly-org` is ...
* `github-org` is ...


(If there are multiple flavors of this command, you could just have a few code blocks like this to explain each flavor separately - each flavor would have different flags)

File: docs/cloud/satellites/gha-runners.md [hunk 3]
```diff
@@ -1,66 +1,48 @@
...
 NAME:
-earthly github add - Add GHA integration
+earthly github add - Add GitHub Actions integration
 
 USAGE:
 earthly github add --org <org> --gh-org <github_organization> [--gh-repo <github_-repo>] --gh-token <github_token>
 
 DESCRIPTION:
-This command sets the configuration to create a new GitHub-Earthly integration, to trigger satellite builds from GHA (GitHub Actions).
+This command sets the configuration to create a new GitHub-Earthly integration, to trigger satellite builds from GitHub Actions (GitHub Actions).
```
Comment (vladaionescu) [inline docs/cloud/satellites/gha-runners.md:45]:
```suggestion
This command sets the configuration to create a new GitHub-Earthly integration, to trigger satellite builds from GitHub Actions.
```

File: docs/cloud/satellites/gha-runners.md [hunk 4]
```diff
@@ -72,71 +54,59 @@ OPTIONS:
 --help, -h        show help
 ```
 
-## Satellite configuration
-This feature is currently disabled by default for satellites. It must be enabled in a per-satellite basis as follows:
+### 4. Configure a satellite 
 
-### Managed (Earthly Cloud) Satellites
+This feature is both available for self-hosted and Earthly-Cloud satellites, but in both cases is disabled by default. 
+It must be enabled in a per-satellite basis as follows:
```
Comment (vladaionescu) [inline docs/cloud/satellites/gha-runners.md:60]: Simplify...

```suggestion
This feature needs to be enabled during satellite creation to be able to use it.
```

File: docs/cloud/satellites/gha-runners.md [hunk 5]
```diff
@@ -72,71 +54,59 @@ OPTIONS:
...
+
+#### Earthly-Cloud satellites
 Launch the satellite with the `enable-gha-runner` [feature-flag](https://docs.earthly.dev/earthly-cloud/satellites/managing#changing-feature-flags) enabled.
 ```
 earthly satellite --feature-flag enable-gha-runner <satellite-name>
 ``` 
 
-### Self-hosted satellites
+#### Self-hosted satellites
 To enable the GH runner for a self-hosted satellite, just set this environment entry when launching it:
```
Comment (vladaionescu) [inline docs/cloud/satellites/gha-runners.md:69]:
```suggestion
To enable the GH runner for a self-hosted satellite, set this environment entry when launching it:
```

**Review by vladaionescu — COMMENTED** [review]
Looking pretty good! A few more minor comments...

File: docs/cloud/satellites/gha-runners.md [hunk 6]
```diff
@@ -1,142 +1,103 @@
...
 
-## Satellite configuration
-This feature is currently disabled by default for satellites. It must be enabled in a per-satellite basis as follows:
+This feature needs to be enabled during satellite creation to be able to use it.
 
-### Managed (Earthly Cloud) Satellites
+#### Earthly-Cloud satellites
 Launch the satellite with the `enable-gha-runner` [feature-flag](https://docs.earthly.dev/earthly-cloud/satellites/managing#changing-feature-flags) enabled.
 ```
 earthly satellite --feature-flag enable-gha-runner <satellite-name>
```
Comment (vladaionescu) [inline docs/cloud/satellites/gha-runners.md:56]: Are you missing the word `launch` in this command?

Comment (idelvall) [reply docs/cloud/satellites/gha-runners.md]: good catch!

**Review by vladaionescu — APPROVED** [review]

File: docs/cloud/satellites/gha-runners.md [hunk 7]
```diff
@@ -1,142 +1,103 @@
-# Satellites as GHA runners (**experimental**)
+# Satellites as GitHub Actions runners (**experimental**)
```
Comment (vladaionescu) [inline docs/cloud/satellites/gha-runners.md:1]: Note that this title doesn't actually show up in GitBook (it uses the title in the SUMMARY.md file).

I would maybe add a warning box at the top to say that this is experimental.

Comment (idelvall) [reply docs/cloud/satellites/gha-runners.md]: done

**Review by vladaionescu — APPROVED** [review]

**Pull request merged**